### ISSUE TITLE
To Support Myanmar Unicode Font

### ISSUE THREAD

**Issue by saturngod** [gravitystorm/openstreetmap-carto#1538]
Currently open street map is using Padauk font and it's only for Unicode 5.1. If it's possible , please support the 5.2 ready Myanamar unicode font. 

Unicode 5.2 support myanmar fonts are
- Mon3 ( https://code.google.com/p/mon2uni/downloads/detail?name=mm3anonta_1%20Update%202013.ttf&can=2&q= )
- Tharlon ( https://www.google.com/fonts/earlyaccess , http://fonts.gstatic.com/ea/tharlon/v3/download.zip ) .

From https://www.openstreetmap.org/#map=14/19.7501/96.1353 , it's showing wrong rendering for Nay Pyi Daw (နေပြည်တော်)

![naypidaw](http://cl.ly/au5A/Screen%20Shot%202015-05-08%20at%2012.21.53%20pm.png)

Text should be render like following

![correct](http://cl.ly/atux/Screen%20Shot%202015-05-08%20at%2012.23.48%20pm.png)

**Comment by pnorman** [comment]
This may be a duplicate of #392, which is not a font issue. That issue is waiting on @Mapnik 3 to be released, which is out of our hands.

**Comment by pnorman** [comment]
Appears to be a duplicate.
